### PR TITLE
External build fix for not linux platforms

### PR DESCRIPTION
**Pull request by seanshpark**
Added generic build for not linux(embedded) platforms as nuttx has be removed by #330

JerryScript-DCO-1.0-Signed-off-by: SaeHie Park saehie.park@samsung.com

### CODE REVIEW

**Comment by seanshpark** [comment]
@egavrin , Jenkins reports with `make-precommit-cross-toolchain_linux_armv7l-el` says
```
No identified problem
No problems were identified. If you know why this problem occurred, please add a suitable Cause for it.
```
What can I do?

**Comment by seanshpark** [comment]
#347

File: CMakeLists.txt [hunk 1]
```diff
@@ -256,6 +266,9 @@ project (Jerry CXX C ASM)
     # stm32f4
      set(SOURCE_JERRY_STANDALONE_MAIN_MCU_STM32F4 main-mcu.cpp)
 
+   # EXTERNAL dummy
+    set(SOURCE_JERRY_STANDALONE_MAIN_EXTERNAL main-external.cpp)
```
Comment (ruben-ayrapetyan) [inline CMakeLists.txt]: @seanshpark, maybe we shouldn't add the file, as external application provides its own `main`. So, jerryscript, in the case, is just linked to some another application.

**Comment by seanshpark** [comment]
@ruben-ayrapetyan , thank you for your help. this PR is for both #342 and #344

**Comment by ruben-ayrapetyan** [comment]
Looks good to me.

**Comment by egavrin** [comment]
@seanshpark please change Signed-off order for "External build fix", otherwise precommit will fail:
```
Added generic build for none linux platforms, as nuttx has be removed by #330

JerryScript-DCO-1.0-Signed-off-by: Ruben Ayrapetyan r.ayrapetyan@samsung.com
JerryScript-DCO-1.0-Signed-off-by: SaeHie Park saehie.park@samsung.com
```

**Comment by galpeter** [comment]
lgtm

**Comment by seanshpark** [comment]
rebased

**Pull request merged**